tasks/download.yml

```
@@ -3,6 +3,7 @@
   get_url:
     url: "{{ jira_download_url }}"
     dest: "{{ jira_app_dir }}/jira.tar.gz"
+    tmp_dest: "{{ tmp_dest }}"
```

I would prefer a variable with application name in it (and a default value (in the same partition than destination file to avoid a +/- 1Go copy)), like:
```suggestion
    tmp_dest: "{{ jira_tmp_dest | default(jira_app_dir) }}"
```